### PR TITLE
Configurable minimum number of peer to broadcast

### PR DESCRIPTION
**Pull request by limebell**
SSIA

### CODE REVIEW

File: Libplanet/Net/SwarmOptions.cs [hunk 1]
```diff
@@ -75,5 +75,11 @@ public class SwarmOptions
         /// using an approximated value.
         /// </summary>
         public int BranchpointThreshold { get; set; } = 10;
+
+        /// <summary>
+        /// The minimum number to select from routing table when broadcast messages.
+        /// It is 10 by default.
+        /// </summary>
+        public int MinimumBroadcastTarget { get; set; } = 10;
```
Comment (dahlia) [inline Libplanet/Net/SwarmOptions.cs:83]: Shouldn't this be recorded in the changelog?

Comment (limebell) [reply Libplanet/Net/SwarmOptions.cs:83]: I was intending to write a changelog when importing to the main branch.

**Review by longfin — APPROVED** [review]

File: Libplanet/Net/SwarmOptions.cs (hunk 1 above)
Comment (dahlia) [reply Libplanet/Net/SwarmOptions.cs:83]: 👌🏼

**Review by dahlia — APPROVED** [review]

**Comment by limebell** [comment]
I cherrypicked commit https://github.com/planetarium/libplanet/pull/1377/commits/ea22b9fe1d24c7d28d69834bf2c288944ae42504. Please review again @dahlia @longfin

**Review by longfin — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

**Review by greymistcube — APPROVED** [review]

**Pull request merged**